doc/tutorial/05_split_security.md

```diff
@@ -1,49 +1,64 @@
 # Split security model in pgagroal
 
-This tutorial will show you how to split the security model of pgagroal such that
-applications will use a different password than the one used against PostgreSQL.
+This tutorial will show you how to split the security model of `pgagroal`.
+
+The idea is that the pooler can act as a *user proxy* between your application and
+the PostgreSQL instance, so that your application does not need to know the exact password
+to use to connect to PostgreSQL.
+`pgagroal` will authenticate the connection request with its credentials, and then will
+authenticate against PostgreSQL with the correct password.
+
+This *user mapping* is named *frontend users*.
 
 ## Preface
 
-This tutorial assumes that you have an installation of PostgreSQL 10+ and pgagroal.
+This tutorial assumes that you have already an installation of PostgreSQL 10 (or higher) and `pgagroal`.
+
+In particular, this tutorial refers to the configuration done in [Install pgagroal](https://github.com/pgagroal/pgagroal/blob/main/doc/tutorial/01_install.md).
+
 
-See [Install pgagroal](https://github.com/pgagroal/pgagroal/blob/main/doc/tutorial/01_install.md)
-for more detail.
+## Create frontend users
 
-## Create pgagroal_frontend_users.conf
+Frontend users are stored into the `pgagroal_frontend_users.conf` file, that can be managed via the
+`pgagroal-admin` command line tool.
+See [the documentation on frontend users](https://github.com/agroal/pgagroal/blob/master/doc/CONFIGURATION.md#pgagroal_frontend_users-configuration) for more details.
 
-Create the configuration file
+As an example, since the `myuser` has the `mypassword` (created in [Install pgagroal](https://github.com/pgagroal/pgagroal/blob/main/doc/tutorial/01_install.md)), we can *remap* the password for such user to `application_password`. This means that when the user `myuser` will connect to the pooler, it will need to specify the `application_password` (not `mypassword`), then `pgagroal` will connect to PostgreSQL with the couple of credentials `mysuers`/`mypassword`.
+
+To achieve this, as `pgagroal` operating system run the following command:
 
 ```
-pgagroal-admin -f pgagroal_frontend_users.conf -U myuser -P apppass add-user
+pgagroal-admin -f /etc/pgagroal/pgagroal_frontend_users.conf -U myuser -P apppass add-user
 ```
 
 You need a password mapping for each user defined in the `pgagroal_users.conf` configuration file.
 
-(`pgagroal` user)
```

Comment:
Keep the user

Reply:
ok, but again not clear the need for the user